src/pybind/mgr/dashboard/plugins/interfaces.py

```diff
@@ -10,8 +10,8 @@ class CanMgr(Mixin):
 
 
 class CanLog(Mixin):
-    from .. import logger
-    log = logger
+    import logging
+    log = logging.getLogger()
```

Comment:
:+1:  for erasing `CanLog` altogether, as CanLog.log is always the root-logger which is odd.

Reply:
yeah, I was also wondering this morning if I should remove the whole thing. @epuertat can we remove it? With this PR we don't need to mgr instance to access the logger anymore...